Tests/Realm.Tests/SourceGeneration/TestClasses/ManuallyGeneratedClass.cs

```diff
@@ -28,6 +28,7 @@ namespace Realms.Tests.SourceGeneration.TestClasses
 {
 #pragma warning disable CA1507 // Use nameof to express symbol names
 
+    [Ignored]
     public partial class ManualllyGeneratedClass : IRealmObject
```

Comment:
Do we need this class anymore?

Reply:
It was useful to test the proof of concept. It has not been updated so probably we can remove it